libs/entry-components/styles/modules/_default-theme.scss

```diff
@@ -47,6 +47,10 @@ $theme: (
 		),
 		buttons: (
 			icon-size: 48px
+		),
+		checkboxes: (
+			background: null,
+			border: null
```

Comment:
Except for hovering effects, we're not changing border colors at all. I suggest changing this name to border-hover-color so we do not confuse anybody. If behavior changes, so will theme.